docker-compose.yml

```diff
@@ -85,6 +85,7 @@ services:
       FEATURE_GET_UPDATES: 'true'
       FEATURE_ALLOW_HAVE_YOUR_SAY: 'true'
       FEATURE_PROJECT_INFORMATION: 'true'
+      PROJECT_MIGRATION_CASE_REFERENCES: 'EN010009, EN010120, EN013012' # change this to any case ids
```

Comment:
As this is for local testing, I think it makes sense to have a bigger complete list

EN010009, EN010011, EN010102
, EN010116, EN013007, EN013009
,  EN013012, EN013013, EN013014
, EN033005, EN043005, EN043006, EN053001, TR010001, TR023018 ,TR023022, TR023024
, TR033002, tr033005, TR040009

Reply:
https://github.com/Planning-Inspectorate/applications-service/pull/778/commits/e1595cf2ff37c645f5a67606a8add9bc1ecc2eac